src/universe/qual/Self.java

```diff
@@ -19,6 +20,7 @@
 @Documented
 @Retention(RetentionPolicy.RUNTIME)
 @Target({ ElementType.TYPE_PARAMETER, ElementType.TYPE_USE })
-@TargetLocations({TypeUseLocation.RECEIVER})
+@TargetLocations({ TypeUseLocation.RECEIVER })
 @SubtypeOf({ Peer.class })
+@DefaultFor({ TypeUseLocation.RECEIVER })
```

Comment:
@leoliu49 Thanks for these changes and sorry for the delay on getting back.
The PR description seems a bit confusing: isn't this addition the only substantial change? And that seems unrelated to updates in checker-qual, isn't it?
Am I missing what this PR is changing?

I would prefer undoing the whitespace changes and re-ordering of imports and do that with an automatic tool once all updates are merged in.

Reply:
I see -- I don't think that change is necessary at this stage. Thanks.
I'll close this PR.